File: output/gemma4-e4b/avatar/Java/Python/codeforces_379_A.py
import sys

def solve():
    # Read all input lines and convert to integers
    try:
        data = sys.stdin.read().split()
        if not data:
            return
        a = int(data[0])
        b = int(data[1])
    except Exception:
        # Handle case where input might be missing or malformed
        return

    res = a
    while a >= b:
        # Integer division in Python is //
        res += a // b
        
        # Update a: (a // b) + (a % b)
        a = (a // b) + (a % b)
    
    print(res)

solve()

File: output/gemma4-e4b/avatar/Python/Java/codeforces_379_A.java
import java.util.Scanner;

public class codeforces_379_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read the two space-separated integers
        if (!scanner.hasNextLong()) {
            // Handle case where input might be missing or malformed
            return;
        }
        
        long candeleIntere = scanner.nextLong();
        long b = scanner.nextLong();
        
        scanner.close();

        long s = 0;
        long restoSciolte = 0;

        while (candeleIntere > 0 || restoSciolte >= b) {
            // candeleIntere += restoSciolte // b
            candeleIntere += restoSciolte / b;
            
            // restoSciolte %= b
            restoSciolte %= b;
            
            // s += candeleIntere
            s += candeleIntere;
            
            // restoSciolte += candeleIntere % b
            restoSciolte += candeleIntere % b;
            
            // candeleIntere //= b
            candeleIntere /= b;
        }
        
        System.out.println(s);
    }
}